addons/sale_crm/crm_lead.py

```diff
@@ -11,6 +17,30 @@ def _get_sale_amount_total(self, cr, uid, ids, fields, args, context=None):
             res[key] = value
         return res
 
+    def action_view_quotation(self, cr, uid, ids, context=None):
+    # '''
```

Comment:
The good format is
`""" This function ...
... blah blah ...
... blah blah """`